recipes/ckan/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "ckan" %}
+{% set version = "2.8.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ckan/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 1fdc09f4b1ae3a70d84138118c86eee056695235e5be8c42501c8af6b6679686
+  patches:
+    - webob.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+  skip: True  # [win]
+
+requirements:
+  host:
+    - pip
+    - python <3.0
```

Comment:
This is not how other python versions are skipped. See https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=skip#skipping-builds

Reply:
thanks, fixed with py>=35